nflow-explorer-ng/package.json

```diff
@@ -1,47 +1,47 @@
 {
-  "name": "explorer2",
+  "name": "explorer-ng",
   "version": "0.1.0",
   "private": true,
+  "homepage": ".",
   "dependencies": {
-    "@fontsource/roboto": "^4.3.0",
-    "@material-ui/core": "^4.11.4",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.60",
-    "@testing-library/jest-dom": "^5.11.4",
+    "@fontsource/roboto": "^4.5.7",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.11.3",
+    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
-    "@types/d3": "^7.1.0",
-    "@types/http-proxy": "^1.17.6",
-    "@types/jest": "^26.0.15",
-    "@types/lodash": "^4.14.170",
-    "@types/mui-datatables": "^3.7.7",
+    "@types/d3": "^7.4.0",
+    "@types/http-proxy": "^1.17.9",
+    "@types/jest": "^26.0.24",
+    "@types/lodash": "^4.14.182",
+    "@types/mui-datatables": "^3.7.9",
     "@types/node": "^12.0.0",
-    "@types/q": "^1.5.4",
-    "@types/react": "^17.0.6",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-router-dom": "^5.1.7",
-    "chokidar": "^3.5.2",
-    "d3": "^7.1.1",
+    "@types/q": "^1.5.5",
+    "@types/react": "^17.0.45",
+    "@types/react-dom": "^17.0.17",
+    "@types/react-router-dom": "^5.3.3",
+    "chokidar": "^3.5.3",
+    "d3": "^7.4.4",
     "dagre": "^0.8.5",
     "dagre-d3": "^0.6.4",
-    "date-fns": "^2.21.3",
+    "date-fns": "^2.28.0",
     "eslint-config-react-app": "^6.0.0",
-    "final-form": "^4.20.4",
     "husky": "^4.3.8",
-    "lint-staged": "^11.0.0",
+    "lint-staged": "^11.2.6",
     "lodash": "^4.17.21",
-    "moment": "^2.29.1",
-    "mui-datatables": "^3.8.2",
+    "moment": "^2.29.3",
+    "mui-datatables": "^3.8.5",
     "mui-rff": "^3.0.16",
     "node-sass": "^6.0.1",
```

Comment:
`node-sass` is deprecated: https://www.npmjs.com/package/node-sass. Also `node-sass` doesn't build on Apple M1.

`sass` seems be drop-in replacement:

```
    "sass": "^1.52.2"
```

Reply:
This is also done now :+1: